### ISSUE TITLE
Errors with 2.1.4

### ISSUE THREAD

**Issue by icex** [dzcpy/transliteration#108]
Updating in our project to 2.1.4 results in the below errors(while building with webpack). 2.1.3 works fine.

```
ERROR in ./node_modules/transliteration/dist/browser/bundle.umd.min.js
Module not found: Error: Can't resolve '../internals/a-function' in '/Users/bogdan/Sites/cs/node_modules/transliteration/dist/browser'
 @ ./node_modules/transliteration/dist/browser/bundle.umd.min.js 1:264768-264802
 @ ./src/js/rpk-getslug.js
 @ ./src/main.js
 @ multi ./src/main.js ./src/plugins-doc.js

ERROR in ./node_modules/transliteration/dist/browser/bundle.umd.min.js
Module not found: Error: Can't resolve '../internals/advance-string-index' in '/Users/bogdan/Sites/cs/node_modules/transliteration/dist/browser'
 @ ./node_modules/transliteration/dist/browser/bundle.umd.min.js 1:2224-2268
 @ ./src/js/rpk-getslug.js
 @ ./src/main.js
 @ multi ./src/main.js ./src/plugins-doc.js

ERROR in ./node_modules/transliteration/dist/browser/bundle.umd.min.js
Module not found: Error: Can't resolve '../internals/an-object' in '/Users/bogdan/Sites/cs/node_modules/transliteration/dist/browser'
 @ ./node_modules/transliteration/dist/browser/bundle.umd.min.js 1:1551-1584 1:2028-2061
 @ ./src/js/rpk-getslug.js
 @ ./src/main.js
 @ multi ./src/main.js ./src/plugins-doc.js

ERROR in ./node_modules/transliteration/dist/browser/bundle.umd.min.js
Module not found: Error: Can't resolve '../internals/array-for-each' in '/Users/bogdan/Sites/cs/node_modules/transliteration/dist/browser'
 @ ./node_modules/transliteration/dist/browser/bundle.umd.min.js 1:263123-263161 1:265235-265273
 @ ./src/js/rpk-getslug.js
 @ ./src/main.js
 @ multi ./src/main.js ./src/plugins-doc.js

ERROR in ./node_modules/transliteration/dist/browser/bundle.umd.min.js
Module not found: Error: Can't resolve '../internals/array-includes' in '/Users/bogdan/Sites/cs/node_modules/transliteration/dist/browser'
 @ ./node_modules/transliteration/dist/browser/bundle.umd.min.js 1:263268-263306
 @ ./src/js/rpk-getslug.js
 @ ./src/main.js
 @ multi ./src/main.js ./src/plugins-doc.js

ERROR in ./node_modules/transliteration/dist/browser/bundle.umd.min.js
Module not found: Error: Can't resolve '../internals/array-iteration' in '/Users/bogdan/Sites/cs/node_modules/transliteration/dist/browser'
 @ ./node_modules/transliteration/dist/browser/bundle.umd.min.js 1:3959-3998
 @ ./src/js/rpk-getslug.js
 @ ./src/main.js
 @ multi ./src/main.js ./src/plugins-doc.js

ERROR in ./node_modules/transliteration/dist/browser/bundle.umd.min.js
Module not found: Error: Can't resolve '../internals/array-method-has-species-support' in '/Users/bogdan/Sites/cs/node_modules/transliteration/dist/browser'
 @ ./node_modules/transliteration/dist/browser/bundle.umd.min.js 1:4038-4094 1:264184-264240
 @ ./src/js/rpk-getslug.js
 @ ./src/main.js
 @ multi ./src/main.js ./src/plugins-doc.js

ERROR in ./node_modules/transliteration/dist/browser/bundle.umd.min.js
Module not found: Error: Can't resolve '../internals/create-property' in '/Users/bogdan/Sites/cs/node_modules/transliteration/dist/browser'
 @ ./node_modules/transliteration/dist/browser/bundle.umd.min.js 1:264141-264180
 @ ./src/js/rpk-getslug.js
 @ ./src/main.js
 @ multi ./src/main.js ./src/plugins-doc.js

ERROR in ./node_modules/transliteration/dist/browser/bundle.umd.min.js
Module not found: Error: Can't resolve '../internals/descriptors' in '/Users/bogdan/Sites/cs/node_modules/transliteration/dist/browser'
 @ ./node_modules/transliteration/dist/browser/bundle.umd.min.js 1:353-388 1:4419-4454
 @ ./src/js/rpk-getslug.js
 @ ./src/main.js
 @ multi ./src/main.js ./src/plugins-doc.js

ERROR in ./node_modules/transliteration/dist/browser/bundle.umd.min.js
Module not found: Error: Can't resolve '../internals/dom-iterables' in '/Users/bogdan/Sites/cs/node_modules/transliteration/dist/browser'
 @ ./node_modules/transliteration/dist/browser/bundle.umd.min.js 1:265194-265231
 @ ./src/js/rpk-getslug.js
 @ ./src/main.js
 @ multi ./src/main.js ./src/plugins-doc.js

ERROR in ./node_modules/transliteration/dist/browser/bundle.umd.min.js
Module not found: Error: Can't resolve '../internals/export' in '/Users/bogdan/Sites/cs/node_modules/transliteration/dist/browser'
 @ ./node_modules/transliteration/dist/browser/bundle.umd.min.js 1:211-241 1:1382-1412 1:3926-3956 1:4185-4215 1:4629-4659 1:263089-263119 1:263234-263264 1:263586-263616 1:263907-263937 1:264734-264764
 @ ./src/js/rpk-getslug.js
 @ ./src/main.js
 @ multi ./src/main.js ./src/plugins-doc.js

ERROR in ./node_modules/transliteration/dist/browser/bundle.umd.min.js
Module not found: Error: Can't resolve '../internals/fails' in '/Users/bogdan/Sites/cs/node_modules/transliteration/dist/browser'
 @ ./node_modules/transliteration/dist/browser/bundle.umd.min.js 1:725-754 1:1587-1616 1:4325-4354 1:264843-264872
 @ ./src/js/rpk-getslug.js
 @ ./src/main.js
 @ multi ./src/main.js ./src/plugins-doc.js

ERROR in ./node_modules/transliteration/dist/browser/bundle.umd.min.js
Module not found: Error: Can't resolve '../internals/fix-regexp-well-known-symbol-logic' in '/Users/bogdan/Sites/cs/node_modules/transliteration/dist/browser'
 @ ./node_modules/transliteration/dist/browser/bundle.umd.min.js 1:1967-2025
 @ ./src/js/rpk-getslug.js
 @ ./src/main.js
 @ multi ./src/main.js ./src/plugins-doc.js

ERROR in ./node_modules/transliteration/dist/browser/bundle.umd.min.js
Module not found: Error: Can't resolve '../internals/forced-string-trim-method' in '/Users/bogdan/Sites/cs/node_modules/transliteration/dist/browser'
 @ ./node_modules/transliteration/dist/browser/bundle.umd.min.js 1:4740-4789
 @ ./src/js/rpk-getslug.js
 @ ./src/main.js
 @ multi ./src/main.js ./src/plugins-doc.js

ERROR in ./node_modules/transliteration/dist/browser/bundle.umd.min.js
Module not found: Error: Can't resolve '../internals/global' in '/Users/bogdan/Sites/cs/node_modules/transliteration/dist/browser'
 @ ./node_modules/transliteration/dist/browser/bundle.umd.min.js 1:391-421 1:265160-265190
 @ ./src/js/rpk-getslug.js
 @ ./src/main.js
 @ multi ./src/main.js ./src/plugins-doc.js

ERROR in ./node_modules/transliteration/dist/browser/bundle.umd.min.js
Module not found: Error: Can't resolve '../internals/hide' in '/Users/bogdan/Sites/cs/node_modules/transliteration/dist/browser'
 @ ./node_modules/transliteration/dist/browser/bundle.umd.min.js 1:265277-265305
 @ ./src/js/rpk-getslug.js
 @ ./src/main.js
 @ multi ./src/main.js ./src/plugins-doc.js

ERROR in ./node_modules/transliteration/dist/browser/bundle.umd.min.js
Module not found: Error: Can't resolve '../internals/indexed-object' in '/Users/bogdan/Sites/cs/node_modules/transliteration/dist/browser'
 @ ./node_modules/transliteration/dist/browser/bundle.umd.min.js 1:263620-263658
 @ ./src/js/rpk-getslug.js
 @ ./src/main.js
 @ multi ./src/main.js ./src/plugins-doc.js

ERROR in ./node_modules/transliteration/dist/browser/bundle.umd.min.js
Module not found: Error: Can't resolve '../internals/inherit-if-required' in '/Users/bogdan/Sites/cs/node_modules/transliteration/dist/browser'
 @ ./node_modules/transliteration/dist/browser/bundle.umd.min.js 1:460-503
 @ ./src/js/rpk-getslug.js
 @ ./src/main.js
 @ multi ./src/main.js ./src/plugins-doc.js

ERROR in ./node_modules/transliteration/dist/browser/bundle.umd.min.js
Module not found: Error: Can't resolve '../internals/is-array' in '/Users/bogdan/Sites/cs/node_modules/transliteration/dist/browser'
 @ ./node_modules/transliteration/dist/browser/bundle.umd.min.js 1:263978-264010
 @ ./src/js/rpk-getslug.js
 @ ./src/main.js
 @ multi ./src/main.js ./src/plugins-doc.js

ERROR in ./node_modules/transliteration/dist/browser/bundle.umd.min.js
Module not found: Error: Can't resolve '../internals/is-forced' in '/Users/bogdan/Sites/cs/node_modules/transliteration/dist/browser'
 @ ./node_modules/transliteration/dist/browser/bundle.umd.min.js 1:424-457
 @ ./src/js/rpk-getslug.js
 @ ./src/main.js
 @ multi ./src/main.js ./src/plugins-doc.js

ERROR in ./node_modules/transliteration/dist/browser/bundle.umd.min.js
Module not found: Error: Can't resolve '../internals/is-object' in '/Users/bogdan/Sites/cs/node_modules/transliteration/dist/browser'
 @ ./node_modules/transliteration/dist/browser/bundle.umd.min.js 1:263941-263974
 @ ./src/js/rpk-getslug.js
 @ ./src/main.js
 @ multi ./src/main.js ./src/plugins-doc.js

ERROR in ./node_modules/transliteration/dist/browser/bundle.umd.min.js
Module not found: Error: Can't resolve '../internals/is-regexp' in '/Users/bogdan/Sites/cs/node_modules/transliteration/dist/browser'
 @ ./node_modules/transliteration/dist/browser/bundle.umd.min.js 1:615-648
 @ ./src/js/rpk-getslug.js
 @ ./src/main.js
 @ multi ./src/main.js ./src/plugins-doc.js

ERROR in ./node_modules/transliteration/dist/browser/bundle.umd.min.js
Module not found: Error: Can't resolve '../internals/object-assign' in '/Users/bogdan/Sites/cs/node_modules/transliteration/dist/browser'
 @ ./node_modules/transliteration/dist/browser/bundle.umd.min.js 1:244-281
 @ ./src/js/rpk-getslug.js
 @ ./src/main.js
 @ multi ./src/main.js ./src/plugins-doc.js

ERROR in ./node_modules/transliteration/dist/browser/bundle.umd.min.js
Module not found: Error: Can't resolve '../internals/object-define-property' in '/Users/bogdan/Sites/cs/node_modules/transliteration/dist/browser'
 @ ./node_modules/transliteration/dist/browser/bundle.umd.min.js 1:506-552 1:4458-4504
 @ ./src/js/rpk-getslug.js
 @ ./src/main.js
 @ multi ./src/main.js ./src/plugins-doc.js

ERROR in ./node_modules/transliteration/dist/browser/bundle.umd.min.js
Module not found: Error: Can't resolve '../internals/object-get-own-property-names' in '/Users/bogdan/Sites/cs/node_modules/transliteration/dist/browser'
 @ ./node_modules/transliteration/dist/browser/bundle.umd.min.js 1:557-610
 @ ./src/js/rpk-getslug.js
 @ ./src/main.js
 @ multi ./src/main.js ./src/plugins-doc.js

ERROR in ./node_modules/transliteration/dist/browser/bundle.umd.min.js
Module not found: Error: Can't resolve '../internals/object-keys' in '/Users/bogdan/Sites/cs/node_modules/transliteration/dist/browser'
 @ ./node_modules/transliteration/dist/browser/bundle.umd.min.js 1:4255-4290
 @ ./src/js/rpk-getslug.js
 @ ./src/main.js
 @ multi ./src/main.js ./src/plugins-doc.js

ERROR in ./node_modules/transliteration/dist/browser/bundle.umd.min.js
Module not found: Error: Can't resolve '../internals/redefine' in '/Users/bogdan/Sites/cs/node_modules/transliteration/dist/browser'
 @ ./node_modules/transliteration/dist/browser/bundle.umd.min.js 1:690-722 1:1516-1548
 @ ./src/js/rpk-getslug.js
 @ ./src/main.js
 @ multi ./src/main.js ./src/plugins-doc.js

ERROR in ./node_modules/transliteration/dist/browser/bundle.umd.min.js
Module not found: Error: Can't resolve '../internals/regexp-exec' in '/Users/bogdan/Sites/cs/node_modules/transliteration/dist/browser'
 @ ./node_modules/transliteration/dist/browser/bundle.umd.min.js 1:1415-1450
 @ ./src/js/rpk-getslug.js
 @ ./src/main.js
 @ multi ./src/main.js ./src/plugins-doc.js

ERROR in ./node_modules/transliteration/dist/browser/bundle.umd.min.js
Module not found: Error: Can't resolve '../internals/regexp-exec-abstract' in '/Users/bogdan/Sites/cs/node_modules/transliteration/dist/browser'
 @ ./node_modules/transliteration/dist/browser/bundle.umd.min.js 1:2271-2315
 @ ./src/js/rpk-getslug.js
 @ ./src/main.js
 @ multi ./src/main.js ./src/plugins-doc.js

ERROR in ./node_modules/transliteration/dist/browser/bundle.umd.min.js
Module not found: Error: Can't resolve '../internals/regexp-flags' in '/Users/bogdan/Sites/cs/node_modules/transliteration/dist/browser'
 @ ./node_modules/transliteration/dist/browser/bundle.umd.min.js 1:651-687 1:1619-1655 1:4508-4544
 @ ./src/js/rpk-getslug.js
 @ ./src/main.js
 @ multi ./src/main.js ./src/plugins-doc.js

ERROR in ./node_modules/transliteration/dist/browser/bundle.umd.min.js
Module not found: Error: Can't resolve '../internals/require-object-coercible' in '/Users/bogdan/Sites/cs/node_modules/transliteration/dist/browser'
 @ ./node_modules/transliteration/dist/browser/bundle.umd.min.js 1:2173-2221
 @ ./src/js/rpk-getslug.js
 @ ./src/main.js
 @ multi ./src/main.js ./src/plugins-doc.js

ERROR in ./node_modules/transliteration/dist/browser/bundle.umd.min.js
Module not found: Error: Can't resolve '../internals/set-species' in '/Users/bogdan/Sites/cs/node_modules/transliteration/dist/browser'
 @ ./node_modules/transliteration/dist/browser/bundle.umd.min.js 1:757-792
 @ ./src/js/rpk-getslug.js
 @ ./src/main.js
 @ multi ./src/main.js ./src/plugins-doc.js

ERROR in ./node_modules/transliteration/dist/browser/bundle.umd.min.js
Module not found: Error: Can't resolve '../internals/sloppy-array-method' in '/Users/bogdan/Sites/cs/node_modules/transliteration/dist/browser'
 @ ./node_modules/transliteration/dist/browser/bundle.umd.min.js 1:263318-263361 1:263707-263750 1:264876-264919
 @ ./src/js/rpk-getslug.js
 @ ./src/main.js
 @ multi ./src/main.js ./src/plugins-doc.js

ERROR in ./node_modules/transliteration/dist/browser/bundle.umd.min.js
Module not found: Error: Can't resolve '../internals/string-trim' in '/Users/bogdan/Sites/cs/node_modules/transliteration/dist/browser'
 @ ./node_modules/transliteration/dist/browser/bundle.umd.min.js 1:4663-4698
 @ ./src/js/rpk-getslug.js
 @ ./src/main.js
 @ multi ./src/main.js ./src/plugins-doc.js

ERROR in ./node_modules/transliteration/dist/browser/bundle.umd.min.js
Module not found: Error: Can't resolve '../internals/to-absolute-index' in '/Users/bogdan/Sites/cs/node_modules/transliteration/dist/browser'
 @ ./node_modules/transliteration/dist/browser/bundle.umd.min.js 1:264014-264055
 @ ./src/js/rpk-getslug.js
 @ ./src/main.js
 @ multi ./src/main.js ./src/plugins-doc.js

ERROR in ./node_modules/transliteration/dist/browser/bundle.umd.min.js
Module not found: Error: Can't resolve '../internals/to-indexed-object' in '/Users/bogdan/Sites/cs/node_modules/transliteration/dist/browser'
 @ ./node_modules/transliteration/dist/browser/bundle.umd.min.js 1:263662-263703 1:264096-264137
 @ ./src/js/rpk-getslug.js
 @ ./src/main.js
 @ multi ./src/main.js ./src/plugins-doc.js

ERROR in ./node_modules/transliteration/dist/browser/bundle.umd.min.js
Module not found: Error: Can't resolve '../internals/to-integer' in '/Users/bogdan/Sites/cs/node_modules/transliteration/dist/browser'
 @ ./node_modules/transliteration/dist/browser/bundle.umd.min.js 1:2136-2170
 @ ./src/js/rpk-getslug.js
 @ ./src/main.js
 @ multi ./src/main.js ./src/plugins-doc.js

ERROR in ./node_modules/transliteration/dist/browser/bundle.umd.min.js
Module not found: Error: Can't resolve '../internals/to-length' in '/Users/bogdan/Sites/cs/node_modules/transliteration/dist/browser'
 @ ./node_modules/transliteration/dist/browser/bundle.umd.min.js 1:2100-2133 1:264059-264092
 @ ./src/js/rpk-getslug.js
 @ ./src/main.js
 @ multi ./src/main.js ./src/plugins-doc.js

ERROR in ./node_modules/transliteration/dist/browser/bundle.umd.min.js
Module not found: Error: Can't resolve '../internals/to-object' in '/Users/bogdan/Sites/cs/node_modules/transliteration/dist/browser'
 @ ./node_modules/transliteration/dist/browser/bundle.umd.min.js 1:2064-2097 1:4218-4251 1:264806-264839
 @ ./src/js/rpk-getslug.js
 @ ./src/main.js
 @ multi ./src/main.js ./src/plugins-doc.js

ERROR in ./node_modules/transliteration/dist/browser/bundle.umd.min.js
Module not found: Error: Can't resolve '../internals/well-known-symbol' in '/Users/bogdan/Sites/cs/node_modules/transliteration/dist/browser'
 @ ./node_modules/transliteration/dist/browser/bundle.umd.min.js 1:795-836 1:264244-264285
 @ ./src/js/rpk-getslug.js
 @ ./src/main.js
 @ multi ./src/main.js ./src/plugins-doc.js

```

**Comment by ptgolden** [comment]
This is the case with my project bundled with browserify as well

**Comment by jthistle** [comment]
I can confirm this. I am also getting this error.

**Comment by dzcpy** [comment]
It should be fixed in 2.1.5 with #106 now. Please let me know if there's still any issues

**Comment by dzcpy** [comment]
Duplicate issue with #102, closing this
